model/Software/Classes/SoftwareArtifact.md

```diff
@@ -26,4 +26,19 @@ such as an electronic file, a software package, a device or an element of data.
 - purpose
   - type: SoftwarePurpose
   - minCount: 0
-
+- /Core/Licensing/concludedLicense
```

Comment:
The property is for this class, not an external one.
It should be `concludedLicense`, and have a description in /Software/Properties.

If it's not used anywhere else in the Licensing namespace, it can be deleted from there.

The same for all of them.

Reply:
Sorry, just to clarify, does this mean that there wouldn't be any classes or properties defined in the Licensing profile? All of the classes are used to build up what is a License, so would all of them need to be in Software instead of Licensing?

Reply:
this is written within the scope of the Software profile. So you need to qualify values from Licensing. I think correct would be:
```suggestion
- /Licensing/concludedLicense
```

Reply:
Thanks @maxhbr -- I think I've updated it correctly to handle this, grateful if you can let me know if it still looks incorrect after the latest commit.